bip-pqc-bitcoin.mediawiki

```diff
@@ -0,0 +1,74 @@
+
+This proposal aims to:
+- Protect Bitcoin from quantum-enabled attacks by integrating post-quantum cryptographic algorithms.
+- Provide a roadmap for transitioning to quantum-safe cryptographic techniques while maintaining Bitcoin's decentralization and security.
+- Ensure Bitcoin’s resilience in the face of future technological advancements, maintaining trust in its decentralized model.
+
+== Specification ==
+The proposed solution introduces the use of post-quantum cryptography to Bitcoin's transaction signing process. It integrates cryptographic primitives that are resistant to quantum computing-based attacks.
+
+### Proposed Algorithm
+- **Algorithm Choice**: The proposal suggests utilizing lattice-based algorithms such as **Kyber** for key exchange and **NTRU** for signatures as post-quantum alternatives to the traditional ECDSA and ECDH (Elliptic Curve Diffie-Hellman).
+- **Integration with Bitcoin**: This proposal does not aim to replace Bitcoin’s existing cryptography but to introduce hybrid solutions, where PQC algorithms are used alongside traditional ones, offering a gradual transition period.
+- **Quantum-Safe Key Management**: The proposal suggests leveraging a hybrid approach where users can generate Bitcoin addresses using PQC methods, alongside existing methods for backward compatibility during the transition period.
+
+### Implementation in Bitcoin
+- **Key Changes to Bitcoin**: 
+  - Replace existing signature verification with the PQC signature scheme.
+  - Introduce a flag to indicate whether a transaction is using the post-quantum secure method.
+  - Ensure backward compatibility by maintaining ECDSA signatures for users who don’t implement the PQC methods.
+
+### Transition Plan
+- **Phased Rollout**: The implementation of PQC can be rolled out in phases, with soft forks to accommodate the transition.
+- **Backward Compatibility**: Bitcoin will continue to accept ECDSA signatures while supporting the new PQC signatures. This ensures that the network can maintain functionality while gradually migrating towards quantum-resistant cryptographic techniques.
+
+== Reference Implementation ==
+The reference implementation of pqcBitcoin can be found at the following GitHub repository: [pqcBitcoin Reference Implementation](https://github.com/QBlockQ/pqc-bitcoin).
+
+The implementation includes the following:
+- A **PQC-enabled wallet** that supports generating quantum-safe addresses and signing transactions using NTRU-based signatures.
+- A **modified Bitcoin node** that validates both ECDSA and PQC signatures, ensuring compatibility with both old and new addresses during the transition period.
+- **Test vectors** that demonstrate the functionality and security of the proposed cryptographic methods when applied to Bitcoin transactions.
+
+== Rationale ==
```

Comment:
Given that this is the third or fourth PQ proposal in a few months, it would be especially appreciated if the Rationale got into alternative approaches.